tests/unit/test_redis_memory.py

```
@@ -0,0 +1,148 @@
+        self.redis_mock.assert_called_once_with(
+            host="test-host", port=6379, password="test-password", decode_responses=True
+        )
+
+    def test_init_with_default_values(self):
+        """Test initialization with default values when env vars are not set"""
+        self.env_patcher.stop()
+        # Clear existing redis client mock calls
+        self.redis_mock.reset_mock()
+
+        # Create new instance with no env vars
+        memory = RedisMemory()
```

Ruff (0.8.2)

Local variable `memory` is assigned to but never used

Remove assignment to unused variable `memory`

(F841)